### PR TITLE
update to manifest for velero double upgrade in 1.3

### PR DESCRIPTION
**Pull request by cdelatte-hpe**
## Summary and Scope
This will upgrade to 1.6.3 via the platform manifest, then we will need to apply another manifest to do the second upgrade.  this will get us within a  release of current upstream

## Issues and Related PRs

* Resolves CASMPET-5332

### Tested on:

  * Virtual Shasta

### Test description:

- Were the install/upgrade-based validation checks/tests run (goss tests/install-validation doc)?
- Were continuous integration tests run? If not, why?
- Was upgrade tested? If not, why? y
- Was downgrade tested? If not, why? y
- Were new tests (or test issues/Jiras) created for this change?

## Pull Request Checklist

- [ ] Version number(s) incremented, if applicable
- [ ] Copyrights updated
- [ ] License file intact
- [ ] Target branch correct
- [ ] CHANGELOG.md updated
- [ ] Testing is appropriate and complete, if applicable
- [ ] [HPC Product Announcement](https://cray.slack.com/archives/C026TVCSXLH) prepared, if applicable

### CODE REVIEW

File: manifests/velero-1.7-upgrade.yaml [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    - name: csm
+      type: repo
+      location: https://arti.dev.cray.com/artifactory/csm-helm-stable-local/
+    - name: csm-algol60
+      type: repo
+      location: https://artifactory.algol60.net/artifactory/csm-helm-charts/
+  - name: cray-velero
+    source: csm-algol60
+    version: 1.7.1
+    namespace: velero
```
Comment (dborman-hpe) [inline manifests/velero-1.7-upgrade.yaml:39]: Please put a newline at the end of the file.  Perhaps you need to change an editor setting to include the final end-of-file newline?

Comment (cdelatte-hpe) [reply manifests/velero-1.7-upgrade.yaml]: it was a copy/paste issue on my side vs using vscodium.  fixed

File: manifests/velero-1.7-upgrade.yaml [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+# THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR
+# OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE,
+# ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR
+# OTHER DEALINGS IN THE SOFTWARE.
+#
+apiVersion: manifests/v1beta1
+metadata:
+  name: platform
```
Comment (bklei) [inline manifests/velero-1.7-upgrade.yaml:26]: We should change this name -- or it'll clobber the platform manifest run details in the loftsman namespace

Comment (cdelatte-hpe) [reply manifests/velero-1.7-upgrade.yaml]: fixed

**Review by gbaker-hpe — APPROVED** [review]

File: manifests/velero-1.7-upgrade.yaml [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+spec:
+  sources:
+    charts:
+    - name: csm
+      type: repo
+      location: https://arti.dev.cray.com/artifactory/csm-helm-stable-local/
+    - name: csm-algol60
+      type: repo
+      location: https://artifactory.algol60.net/artifactory/csm-helm-charts/
+  - name: cray-velero
```
Comment (rsjostrand-hpe) [inline manifests/velero-1.7-upgrade.yaml:36]: This is not valid yaml/loftsman manifest. 
```bash
$ yq e . velero-1.7-upgrade.yaml
Error: bad file 'velero-1.7-upgrade.yaml': yaml: line 4: did not find expected key
```

The `charts` keys is missing. Should look something like the following:
```yaml
apiVersion: manifests/v1beta1
metadata:
  name: velero-double-jump
spec:
  sources:
    charts:
    - name: csm
      type: repo
      location: https://arti.dev.cray.com/artifactory/csm-helm-stable-local/
    - name: csm-algol60
      type: repo
      location: https://artifactory.algol60.net/artifactory/csm-helm-charts/
  charts:
  - name: cray-velero
    source: csm-algol60
    version: 1.7.1
    namespace: velero
```

**Pull request merged**